osidb/models.py

```diff
@@ -958,6 +958,20 @@ def _validate_cvss_scores_and_nist_cvss_validation(self):
                 "NIST CVSSv3 and RH CVSSv3 scores assigned.",
             )
 
+    def _validate_impact_and_summary(self):
+        """
+        Checks that if impact has MODERATE, IMPORTANT or CRITICAL value set,
+        then summary must not be missing.
+        """
+        if (
+            self.impact in [Impact.MODERATE, Impact.IMPORTANT, Impact.CRITICAL]
+            and not self.summary
+        ):
+            self.alert(
+                "impact_without_summary",
+                f"summary cannot be missing if impact is {self.impact}.",
```

Comment:
nitpick: by the quick check it seems that the alert texts typically start with uppercase letter